static/images/illustrations/illustration-woo-magic-link.svg

```diff
@@ -0,0 +1 @@
11H54.187v-67.3l56.637-43.509 56.637 43.51v67.299z" fill="#fff"/><path d="M163.385 134.222H58.263V71.766l52.561-40.377 52.561 40.377v62.456z" fill="#E0E0E0"/><path d="M128.655 30.455L64.531 47.25l21.38 81.478 64.125-16.794-21.381-81.479z" fill="#7F54B3"/><path d="M144.291 42.834h-66.29v84.232h66.29V42.834z" fill="#F5F5F5"/><path opacity=".05" d="M54.723 71.019l56.637 33.649-56.637 33.65v-67.3z" fill="#000"/><path d="M54.294 71.019l56.637 33.649-56.637 33.65v-67.3z" fill="#fff"/><path opacity=".05" d="M167.139 71.019l-56.637 33.649 56.637 33.65v-67.3z" fill="#000"/><path d="M167.568 71.019l-56.637 33.649 56.637 33.65v-67.3z" fill="#fff"/><path opacity=".05" d="M54.294 137.247l56.637-44.152 56.637 44.152H54.294z" fill="#000"/><path d="M54.294 138.318l56.637-44.152 56.637 44.152H54.294z" fill="#fff"/><path d="M65.804 143.43s-2.845-9.645.534-16.748a15.224 15.224 0 001.234-9.559 25.601 25.601 0 00-1.354-4.486" stroke="#535461" stroke-width="2" stroke-miterlimit="10"/><path d="M68.319 108.149c0 1.183-2.146 4.61-2.146 4.61s-2.145-3.429-2.145-4.61a2.143 2.143 0 012.145-2.144 2.147 2.147 0 012.145 2.144zM71.87 114.699c-.645.992-4.309 2.707-4.309 2.707s.067-4.042.71-5.035a2.145 2.145 0 013.598 2.328zM71.706 125.889c-1.058.531-5.087.165-5.087.165s2.1-3.455 3.156-3.987a2.154 2.154 0 012.43.232 2.14 2.14 0 01.198 3.089 2.144 2.144 0 01-.697.503v-.002zM69.68 134.331c-.952.703-4.985 1.02-4.985 1.02s1.484-3.759 2.437-4.462a2.148 2.148 0 012.915.508 2.142 2.142 0 01-.366 2.934zM63.054 117.633c.858.823 4.803 1.678 4.803 1.678s-.967-3.924-1.817-4.747a2.146 2.146 0 00-3.566 1.586 2.142 2.142 0 00.58 1.485v-.002zM60.81 127.537c1.057.532 5.086.165 5.086.165s-2.1-3.455-3.155-3.987a2.15 2.15 0 00-2.8.988 2.141 2.141 0 00.869 2.836v-.002zM59.952 137.46c.952.703 4.985 1.021 4.985 1.021s-1.484-3.76-2.437-4.463a2.151 2.151 0 00-2.44-.191 2.158 2.158 0 00-1.006 1.358 2.138 2.138 0 00.898 2.275z" fill="#7F54B3"/><path opacity=".5" d="M108.464 46.692H81.862v2.143h26.602v-2.143zM139.357 55.265H81.862v2.143h57.495v-2.143zM139.357 60.838H81.862v2.143h57.495v-2.143z" fill="#7F54B3"/><path opacity=".4" d="M139.357 66.41H81.862v2.144h57.495V66.41zM139.357 71.983H81.862v2.143h57.495v-2.143z" fill="#7F54B3"/><path opacity=".3" d="M114.686 77.556H81.862v2.143h32.824v-2.143zM126.7 83.128H81.862v2.143H126.7v-2.143z" fill="#7F54B3"/></g><defs><linearGradient id="paint0_linear" x1="22.5" y1="75.911" x2="68.415" y2="75.911" gradientUnits="userSpaceOnUse"><stop stop-color="gray" stop-opacity=".25"/><stop offset=".54" stop-color="gray" stop-opacity=".12"/><stop offset="1" stop-color="gray" stop-opacity=".1"/></linearGradient><linearGradient id="paint1_linear" x1="74840.3" y1="85055.4" x2="74840.3" y2="23540.6" gradientUnits="userSpaceOnUse"><stop stop-color="gray" stop-opacity=".25"/><stop offset=".54" stop-color="gray" stop-opacity=".12"/><stop offset="1" stop-color="gray" stop-opacity=".1"/></linearGradient><clipPath id="clip0"><path fill="#fff" d="M0 0h206v156H0z"/></clipPath></defs></svg>
```

Comment:
I copied this SVG directly from the Figma file. I used `svgo` to make it smaller.